### DISCUSSION TITLE
Is there a way to zoom-out camera view in recordings?

### DISCUSSION THREAD

**Discussion by lorenzomartini1** [blakeblackshear/frigate#15235] [Ask A Question]
Is it possible to zoom-out the camera view? It's very difficult to watch the video feed like this.

![image](https://github.com/user-attachments/assets/f7a22c2c-b251-4dac-9cb8-6240eb6790a5)

**Comment by hawkeye217** [comment]
No. Though this is improved in the latest stable version of Frigate, 0.14. You are running 0.13.
